registration-processor-default.properties

```diff
@@ -984,3 +984,5 @@ mosip.registration.processor.lostrid.max-registration-date-filter-interval=30
 
 ##Ability to update contact information as part of LOST UIN
 uingenerator.lost.packet.allowed.update.fields=phone,email,permanentAddress
+logging.level.io.mosip.registration.processor.core.eventbus.KafkaMosipEventBus=DEBUG
+mosip.regproc.health-check.handler-timeout=5000
```

Comment:
can you add comment on the above configs, 
for the first one lets say this is temporary added for debuging.
for the second one lets explain what is the config about and what is the unit of timeout

Reply:
fixed